src/ui/preferences/GeneralSettings.qml

```diff
@@ -41,6 +41,14 @@ QGCView {
 
     QGCPalette { id: qgcPal }
 
+    FileDialog {
```

Comment:
@jaxxzer Actually how about this. Leave this as is. Then add an Issue to convert to new QGCFileDialog control that I will create. Then once I'm done with that, I'll convert this stuff over. That way you don't need to wait for me.